packages/connect-web/src/webextension/README.md

```diff
@@ -128,3 +128,18 @@ Chrome extension requires a special `trezor-usb-permissions.html` file served fr
 This page will be displayed in case where user is using Trezor without `Trezor Bridge` installed and `navigator.usb` is available.
 
 Lastly, you have to place [this](./trezor-usb-permissions.js) Javascript file into your `vendor/` directory. This directory could be changed, but then you need to remember to change script src accordingly inside `trezor-usb-permissions.html` file.
+
+# TrezorConnect Support Matrix
+
+The table below details the support for different environments by TrezorConnect for integrating Trezor devices, including the use of WebUSB and the need for Trezor Bridge.
+
+| Environment                   | Chrome | Firefox | Chrome Android | Firefox Android | Safari | Edge | Notes                                                                   |
+| ----------------------------- | :----: | :-----: | :------------: | :-------------: | :----: | :--: | ----------------------------------------------------------------------- |
+| Web (WebUSB)                  |   ✓    |    ✗    |       ✗        |        ✗        |   ✗    |  ✓   | WebUSB is fully supported where indicated.                              |
```

Comment:
doesn't chroe android support webusb? Or am I reading the table wrong?

Reply:
In the table I am saying that Trezor Connect as a library for 3rd party developers does NOT support Chrome Android since we are giving the screen "Smartphones not supported yet" in popup in that situations. 

I think it should be technically possible since it works for suite web, but the current status is not working by design.